perception/radar_object_tracker/models.md

```diff
@@ -74,3 +74,25 @@ v_{k+1} &= v_k \\
 \omega_{k+1} &= \omega_k
 \end{align}
 $$
+
+## Noise filtering
+
+Radar sensors often have noisy measurement. So we use the following filter to reduce the false positive objects.
+
+The figure below shows the current noise filtering process.
+
+![noise_filter](image/noise_filtering.drawio.svg)
+
+### minimum range filter
+
+In most cases, Radar sensors are used with other sensors such as LiDAR and Camera, and Radar sensors are used to detect objects far away. So we can filter out objects that are too close to the sensor.
+
+### lanelet based filter
+
+With lanelet map information, We can filter out false positive objects that are not likely important obstacles.
+
+We fukter out objects that satisfy the following conditions:
```

Comment:
```suggestion
We filter out objects that satisfy the following conditions:
```

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/4935/commits/2f0c534887bb3b5c38b8bd776f4e675dbc81cf34.